core-services/image-mirroring/openshift/mapping_origin_4_15

```diff
@@ -128,7 +128,6 @@ registry.ci.openshift.org/origin/4.15:descheduler quay.io/openshift/origin-desch
 registry.ci.openshift.org/origin/4.15:docker-builder quay.io/openshift/origin-docker-builder:4.15 quay.io/openshift/origin-docker-builder:4.15.0 quay.io/openshift/origin-docker-builder:latest
 registry.ci.openshift.org/origin/4.15:docker-registry quay.io/openshift/origin-docker-registry:4.15 quay.io/openshift/origin-docker-registry:4.15.0 quay.io/openshift/origin-docker-registry:latest
 registry.ci.openshift.org/origin/4.15:dpu-network-operator quay.io/openshift/origin-dpu-network-operator:4.15 quay.io/openshift/origin-dpu-network-operator:4.15.0 quay.io/openshift/origin-dpu-network-operator:latest
-registry.ci.openshift.org/origin/4.15:driver-toolkit quay.io/openshift/origin-driver-toolkit:4.15 quay.io/openshift/origin-driver-toolkit:4.15.0 quay.io/openshift/origin-driver-toolkit:latest
```

Comment:
Who is using those images?
DTK's customers are usually getting the image using `oc adm release info ... --image-for driver-toolkit` and getting an image from `quay.io/openshift-release-dev/...`

Reply:
Before merging this PR, we need to make sure that OKD's users running `oc adm release info ... --image-for driver-toolkit` won't get those images. If this is the case then we shouldn't remove them.

Reply:
Customers shouldn't be using any OKD images unless they specifically want to do that. OKD won't have the driver-toolkit going forward, unless OKD-variant specific builds of that image are set up.